### ISSUE TITLE
break list results misaligned with function docs in file

### ISSUE THREAD

**Issue by thecb4** [yonaskolb/Beak#23]
beak.swift file

```swift
/**
 dummy
*/
public func dummy() throws {}

/**
 Releases the product
*/
public func release(version: String) throws {
    // implementation here
    print("version \(version) released! for platform \(platform)")
}

/**
 Installs the product
*/
public func install() throws {
    // implementation here
    print("installed")
}

/**
 Deletes the product
*/
public func delete() throws {
    // implementation here
    print("deleted")
}
```

```zsh
Functions:

  dummy: Releases the product
  release: Installs the product
  install: Deletes the product
  delete: 

```

would have expected:

```zsh
Functions:

  dummy: dummy
  release: Releases the product
  install: Installs the product
  delete: Deletes the product

```

same results regardless of using /** */ or ///

**Comment by yonaskolb** [comment]
I wonder if this is related to https://github.com/yonaskolb/Beak/pull/21
I haven't had a chance to release this yet.
Are you able to try this out from master? If you have Mint installed you can do
```
mint run yonaskolb/beak@master beak list
```

**Comment by thecb4** [comment]
yep - exactly #23. Just tried 

```zsh
mint run yonaskolb/beak@master beak list
```

It works

**Comment by yonaskolb** [comment]
Ok cool. I'll see if I can get a new release out later today 👍

**Comment by yonaskolb** [comment]
Released in 0.3.5
